lib/tasks/defaults.rake

```diff
@@ -1,7 +1,7 @@
 namespace :defaults do
   desc "Create default roles by name"
   task(create_roles: :environment) do
-    %w[archivist no_resets opendoors protected_user tag_wrangler translation_admin translator official].each do |role|
+    %w[archivist no_resets opendoors protected_user tag_wrangler translator official].each do |role|
```

Comment:
Unsure if this is equivalent to adding a rake task to remove "translation_admin" from the roles table

Reply:
This will prevent it from being created, but it won't remove it if it already exists.